cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/batch/AbstractBatchContext.java

```diff
@@ -92,6 +92,17 @@ public T call() throws Exception {
   }
 
   @Override
+  public <T extends Dataset> T getDataset(final String namespace, final String name)
```

Comment:
If that is the case, then all the plugins that take a dataset name as their configuration need to be extended to also accept a namespace. I believe currently none of them do.